README.md

```diff
@@ -47,3 +47,9 @@ neuen CSV-Datei in target ausgegeben.
 Hiermit wird, mithilfe der Koordinaten, eine Weltkarte, inklusive markierter Strandorte, erstellt.
 
 Das shellscript führt anschließend Inkscape aus, um den relevanten Bereich vergrößert zu exportieren.
+
+#### for docker 
+#### build image
+docker build -t automate_image -f collaborators-worldmap/docker/Dockerfile .
```

Comment:
It is unclear, from which directory this command should be executed..
Add more detailed documentation. E.g. open a command line shell in the parent folder.

Even better, don't require the command to be executed from the parent directory. E.g. copy the required files from outside beforehand to the target directory. After that, docker can be started just from the main directory.